torchrec_dlrm/requirements.txt

```diff
@@ -0,0 +1,3 @@
+fbgemm-gpu
```

Comment:
this is fine for now, but let's add a pinned version soon (we have our next release for fbgemm-gpu and TorchRec coming out Friday this week)